### ISSUE TITLE
Access-Control-Allow-Credentials: True causing proxied requests to fail

### ISSUE THREAD

**Issue by msa0311** [Rob--W/cors-anywhere#330]
cors-anywhere does not work if credentials are passed in the requested:
If "Access-Control-Allow-Credentials" is "true" -->  "Access-Control-Allow-Origin" cannot be "*"

> Access to XMLHttpRequest at 'http://localhost:8080/http://xyz' from origin 'http://localhost:12345' has been blocked by CORS policy: The value of the 'Access-Control-Allow-Origin' header in the response must not be the wildcard '*' when the request's credentials mode is 'include'. The credentials mode of requests initiated by the XMLHttpRequest is controlled by the withCredentials attribute.

**Comment by Rob--W** [comment]
Duplicate of #36 among many others.

In short, credentials are not supported for security reasons. See also https://github.com/Rob--W/cors-anywhere/pull/148#issuecomment-452465262
